Snapshot terminology update (#850)
chromaui/chromatic-docs: src/content/turbosnap/turbosnap.mdx

Reviewer: ```suggestion
Certain code changes have the potential to impact all stories. To avoid false positives, we re-capture everything in the following situations:
```

Reviewer: I don't know what's going on with this suggestion by I am suggesting changing "re-test" to "re-capture"

Reviewer 2: Do we spell it re-captured or recaptured?

Author: I prefer `re-captured`

# Introduction to TurboSnap

> TurboSnap is an advanced Chromatic feature that speeds up [UI Tests](/docs#test-how-uis-look--function). It analyzes your project's Git history and builder's [dependency graph](https://webpack.js.org/concepts/dependency-graph/) to identify which components and their dependencies have changed. It intelligently snapshots only the stories associated with those changes. For the rest, it copies over the snapshots from baselines that didn't change.

Reviewer: ```suggestion
TurboSnap is an advanced Chromatic feature that speeds up [UI Tests](/docs#test-how-uis-look--function). It analyzes your project's Git history and builder's [dependency graph](https://webpack.js.org/concepts/dependency-graph/) to identify which components and their dependencies have changed. It intelligently captures snapshots only of the stories associated with those changes. For the rest, it copies over the snapshots from baselines that didn't change.
```

I would suggest using "snapshot" as a verb should be avoided for (hopefully) obvious reasons.

Reviewer: I found a few other instances of this. I don't know if it's the biggest deal but I'd be inclined to change them as you find them.

Reviewer 2: Agree, can we not use snapshot as a verb. Instead use something like "capture a snapshot". This occurs in another couple of places as well in the PR.

Author: Yea, I thought I caught these. I'm going to update the QA skill to check for these too.

## Pricing

Each snapshot incurs billed snapshots: captured snapshots cost 1 billed snapshot and turbosnaps cost 0.2 billed snapshots. The billed snapshot count is used to calculate your monthly bill.

> ### How turbosnaps are billed

Reviewer 2: ```suggestion
### How TurboSnaps are billed
```

I'm having trouble making sense of when to capitalize this. Should we always capitalize this to make it simpler on ourselves?

Author: I think it's pretty straightforward, TurboSnap when talking about the feature and 'turbosnaps' for the snapshot. There will be automated QA so, it'll be easy to catch.

> By enabling TurboSnap, Chromatic checks for stories with no associated code changes. Instead of capturing new snapshots, it copies snapshots from existing baselines. This uses fewer infrastructure resources, and we pass these savings on to you by billing each turbosnap at 1/5th the cost of a captured snapshot.

Reviewer 2: ```suggestion
By enabling TurboSnap, Chromatic checks for stories with no associated code changes. Instead of capturing new snapshots, it copies snapshots from existing baselines. This uses fewer infrastructure resources, and we pass these savings on to you by billing each turbosnap at 1/5th the cost of a captured snapshot (`0.2`).
```

> The remaining stories on that build are captured as usual and billed at the captured-snapshot rate (1 billed snapshot each).

Reviewer 2: ```suggestion
The remaining stories on that build are captured as usual and cost 1 billed snapshot each.
```
